Aptoide/aptoide-client-v8
# [ASV-964-963-965] [ASV-990] MTP for App of the week

**What does this PR do?**

 
  Adds the App of the Week (Editorial). (All the requests need to display the view are made in this ticket also)
  Color pick for the Game of the Week card
  Install card animation
  StatusBar transparent

**Database changed?**

   No

**Where should the reviewer start?**

EditorialFragment.java

**How should this be manually tested?**

Go to the view and check for layout errors. Monkey test with the view. Check if request are being done correctly

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-964
https://aptoide.atlassian.net/browse/ASV-841
https://aptoide.atlassian.net/browse/ASV-990
https://github.com/Aptoide/aptoide-client-v8/pull/691
https://github.com/Aptoide/aptoide-client-v8/pull/692

**Code Review Checklist**
